MMBN3Client.py

```diff
@@ -290,12 +290,9 @@ async def gba_sync_task(ctx: MMBN3Context):
 
 
 async def run_game(romfile):
-    options = Utils.get_options().get("mmbn3_options", None)
-    if options is None:
-        auto_start = True
-    else:
-        auto_start = options.get("rom_start", True)
-    if auto_start:
+    from settings import get_settings
+    auto_start = get_settings().mmbn3_options.rom_start
```

Comment:
i know you implemented literally the code i told you to but i was reminded between making that comment and now that this is the better way for world-specific code to interact with the settings api
```suggestion
    from worlds.mmbn3 import MMBN3World
    auto_start = MMBN3World.settings.rom_start
```